app/models/integration_collection_mappings/netsuite_collection_mapping.rb

```diff
@@ -3,6 +3,35 @@
 module IntegrationCollectionMappings
   class NetsuiteCollectionMapping < BaseCollectionMapping
     settings_accessors :tax_nexus, :tax_type, :tax_code
+    settings_accessors :currencies
+
+    validate :currency_mapping_format
+
+    private
+
+    def currency_mapping_format
+      # Other mapping_types shouldn't have currencies, but if they do, we validate the format
```

Comment:
Shouldn't we ensure abscence instead ?

Reply:
I remember why I did this.
It made testing easier (in the short term) in gql mutaitons ^^